docs/tributech_agent/sources/ads_source.mdx

```diff
@@ -46,7 +46,7 @@ We can now add by right clicking the Device Edge entry a new Beckhoff TwinCat AD
 
 ![**"Add ADS Source**](../img/node-source-ads-add.png)
 
-In order to connect to the ADS Server we need to configure the ADS Server settings. The following table contains the description for each setting for our MockClient (later we will describe how to setup a [MockServer](#mock-server) if you don't have a real ADS Server available):
+In order to connect to the ADS Server we need to configure the ADS Server settings. The following table contains the description for each setting for our MockClient. The information needed to set up our MockClient also can also be found in form of a table here: [MockServer](#mock-server). In this section we also describe how to setup a MockServer if you don’t have your own real ADS Server:
```

Comment:
2x also and a bit complicated sentence, I would suggest to simple write "can be found here"